app/services/messaging/alpha_sms/sms.rb

```diff
@@ -0,0 +1,44 @@
+class Messaging::AlphaSms::Sms < Messaging::Channel
+  def self.communication_type
+    Communication.communication_types[:sms]
+  end
+
+  def send_message(recipient_number:, message:, &with_communication_do)
+    track_metrics do
+      create_communication(
+        recipient_number,
+        message,
+        send_sms(recipient_number, message)["data"]["request_id"],
+        &with_communication_do
+      )
+    end
+  end
+
+  private
+
+  def send_sms(recipient_number, message)
+    Messaging::AlphaSms::Api.new.send_sms(
```

Comment:
We can make class methods that delegate to these instance methods. That's more consistent with our other SMS channel interfaces I think.

Reply:
I don't find the class methods very helpful tbh. In this case for example, the `.new` explicitly tells that an instance of the API client has been instantiated. I was a bit iffy about the channel's interface too so thinking I'd leave this be😅

Reply:
Coolbeans, I don't feel strongly about this, just thought of saving a few keystrokes, nothing major.